Android/app/src/main/res/layout/content_my_trip_info.xml

```diff
@@ -105,22 +98,49 @@
                     android:padding="10dp"
                     android:paddingBottom="13dp"
                     android:src="@drawable/ic_edit_black_24dp" />
+
+                <TextView android:id="@+id/label_trip_date"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_below="@+id/trip_name"
+                    android:visibility="gone"
+                    android:text="Trip Date: "
```

Comment:
dont add constant string in strings.xml file

Reply:
Can I add this text in string.xml file or not.

Reply:
Yes. add constants to strings.xml file. Not directly in layout files.